packages/did-manager/src/did-discovery-provider.ts

```diff
@@ -0,0 +1,22 @@
+import { IAgentContext, IDIDManager } from '@veramo/core';
+import { AbstractDidDiscoveryProvider, IDIDDiscoverMatch, IDIDDiscoveryDiscoverDidArgs } from '@veramo/did-discovery'
+
+export class AliasDiscoveryProvider implements AbstractDidDiscoveryProvider {
```

Comment:
As stated earlier, I think this implementation should be moved to the `did-discovery` package.